src/main/java/frc/robot/subsystems/Limelight.java

```diff
@@ -59,70 +57,63 @@ public double getPitch() {
     }
 
     public int getLightMode() {
-        return lightMode.getNumber(0)
-            .intValue();
+        return lightMode.getNumber(0).intValue();
     }
 
-    public void setLEDMode(LEDMode ledMode) {
-        this.lightMode.setNumber(ledMode.intValue());
+    public void setLEDMode(int ledMode) {
+        this.lightMode.setNumber(ledMode);
```

Comment:
this actually seems like a backward change?

Reply:
It is a temporary backward change because the LEDMode enum (and other limelight enum) code actually doesn't exist yet. We will probably add it in another PR shortly. This basically just reverted the remnants of it so the PR could be focused on LED stuff. It seems as though I attempted to revert it before to get the 2021 repo set up.